pkg/cosign/cosign.go

```diff
@@ -116,17 +125,11 @@ func FetchAttestations(imageRef string, key []byte, repository string) ([]map[st
 
 	cosignOpts := &cosign.CheckOpts{
 		//RootCerts:            fulcio.GetRoots(),
```

Comment:
Looks like we an uncomment this - was left over, not part of your changes.

Reply:
> This looks great! Just one minor request to add the same logic to handle the alternate repository into FetchAttestations.

Thanks! Just implemented this feature and fixed the `fmt` issues.